cfgov/v1/wagtail_hooks.py

```diff
@@ -67,15 +71,26 @@ def configure_page_revision(page, is_sharing, is_live):
 
 def flush_akamai(page, is_live):
     if is_live and settings.ENABLE_AKAMAI_CACHE_PURGE:
-        from publish_eccu.publish import publish as akamai_cache_reset
-
-        url_paths = [page.url_path.replace('cfgov/', '')]
-        if url_paths[0] == '/':
-            is_home_page = True
-        else:
-            is_home_page = False
-
-        akamai_cache_reset(url_paths, invalidate_root=is_home_page, user_email=page.owner.email)
+        akamai_user = os.environ.get('AKAMAI_USER')
+        akamai_pw = os.environ.get('AKAMAI_PASSWORD')
+        akamai_object_id = os.environ.get('AKAMAI_OBJECT_ID')
```

Comment:
The case I was trying to avoid was where we think we've flushed, but for some reason (the variables aren't set, or Akamai returns an error) things fail. If all we do is log that error, then we'd never know unless we happened to check the log If the code raised an exception if something isn't set up or things go wrong, it'd be an easier way to tell because the caller or upstream request (Wagtail editor) would see the error happen.